index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <div>
+                <a href="#">My Logo</a>
+            </div>
+            <div class="hamburger_menu"><img src="images/Union.png" alt=""></div>
+        </nav>
+    </header>
+    <main>
+        <section class="base_section">
+            <h1>
+                I’m Chase Glad to see you!
+            </h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <h2>
+                LET’S CONNECT
+            </h2>
+            <ul>
+                <li><a style="width:19.99px; height: 19.96px;" href="#"><img src="images/socials-1.svg" alt=""></a></li>
+                <li><a style="width:20px; height: 20px;" href="#"><img src="images/socials-2.svg" alt=""></a></li>
+                <li><a style="width:20px; height: 16px;" href="#"><img src="images/socials-3.svg" alt=""></a></li>
+                <li><a style="width:17.99px; height: 20px;" href="#"><img src="images/socials-4.svg" alt=""></a></li>
+            </ul>
+    
+        </section>
+
+    </main>
+
+    
```

Comment:
[Optional] Please consider using an indentation of two spaces in all your documents. This is a convention and also best practice.

Reply:
@Kamba56 Thank you!